_CoqProject

```diff
@@ -69,10 +65,6 @@ theories/Data/Unit.v
 theories/Data/Vector.v
 theories/Data/Z.v
 
-theories/Data/POption.v
-theories/Data/PList.v
-theories/Data/PPair.v
```

Comment:
Sorry, when I said "remove the p-versions of things" I was thinking about `pbind`, `pret`, etc. The universe polymorphic definitions here, are, I believe, still useful.

Reply:
Now I've put them back.